tests/drivers/counter/counter_basic_api/src/test_counter.c

```diff
@@ -174,7 +181,12 @@ void test_set_top_value_with_alarm_instance(const char *dev_name)
 	k_sem_reset(&top_cnt_sem);
 
 	dev = device_get_binding(dev_name);
-	top_cfg.ticks = counter_us_to_ticks(dev, COUNTER_PERIOD_US);
+	if (strcmp(dev_name, "RTC_0") == 0) {
+		counter_period_us = COUNTER_PERIOD_US_VAL;
+	} else {
+		counter_period_us = 20000;
```

Comment:
tough seeing a hard coded value like this in source.. maybe a comment on why for a future viewer of this code?

Reply:
ok

Reply:
this value is the default test range for pit to count, RTC drivers need calculated by (USEC_PER_SEC * 2U), so if one platform has two counter driver RTC and PIT then, this will be conflicts, so I separarted them by the dev_name.